### ISSUE TITLE
Collapsible code editors in panel options

### ISSUE THREAD

**Issue by PhilippDehner** [gapitio/gapit-htmlgraphics-panel#179]
**What would you like to be added**:

The code editors in the panel options should be collapsible.

**Why is this needed**:

* Many times I have to edit the html and the onInit. In this case I have to scroll many times over the onRender block.
* If I want to scroll over the code blocks, I have to move my mouse outside of the code blocks to not scroll in the code.

**Comment by PhilippDehner** [comment]
Or like the aidanmountford-html-panel, here are the html (html and css) and events (onRender and onInit) grouped in two seperate groups

**Comment by ZuperZee** [comment]
Would you also like the options for onInit to be in the same collapsible section as the onInit code?
![image](https://github.com/gapitio/gapit-htmlgraphics-panel/assets/39485579/3654ec4d-9a20-4f9c-a0d4-e40933045097)

**Comment by PhilippDehner** [comment]
@ZuperZee Yes, I would do it like that. But maybe it is better to just collapse the monaco editor...?

**Comment by PhilippDehner** [comment]
> The PR above isn't exactly what you asked for, but (hopefully) solves the problem with the scrolling.

Actually I forked this repo and did the same, I set my wanted height hard coded.

Thanks, that's a good solution!
